docs(webhosting): docs webhosting (#1204)
scaleway/docs-content: managed-services/webhosting/how-to/manage-webhosting.mdx

---
meta:
  title: How to manage a Webhosting
  description: This page explains how to manage a Webhosting from the Scaleway console.
content:
  h1: How to manage a Webhosting
  paragraph: This page explains how to manage a Webhosting from the Scaleway console.
tags: webhosting 
dates:
  validation: 2022-11-07
  posted: 2022-11-07
categories:
  - managed-services
---

> You can access the cPanel interface from the Scaleway console to mange your Webhosting in full autonomy.

Reviewer: ```suggestion
You can access the cPanel interface from the Scaleway console to autonomously mange your Webhosting.
```

<Macro id="iam-requirements" />

<Message type="requirement">
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
  - You have [registered a domain name](/network/domains-and-dns/how-to/register-internal-domain/) at Scaleway or another registrar
  - You have a Webhosting service
</Message>

1. Click **Webhosting** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu. The **Webhosting** overview page displays.

> 2. Click <icon name="seeMore" /> or the domain name of the webhosting you want to configure. The **Hosting information** page displays.

Author: ```suggestion
2. Click <Icon name="seeMore" /> or the domain name of the webhosting you want to configure. The **Hosting information** page displays.
```

3. Click **Dashboard** next to **Link to cPanel** to open the cPanel interface in a new tab.
    <Lightbox src="scaleway-webhosting-cpanel.webp" alt="" />
    <Message type="tip">

>         Alternatively you can reach cPanel via the **cPanel** button in the top right corner of the **Hosting information** section.

Reviewer: ```suggestion
        Alternatively you can reach cPanel via the **cPanel** button in the top right corner of the **Hosting information** section.
```
```suggestion
        Alternatively, you can reach cPanel via the **cPanel** button in the top right corner of the **Hosting information** section.
```
